File: app/src/main/java/com/openclassrooms/realestatemanager/models/Address.java
package com.openclassrooms.realestatemanager.models;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Address {

    @PrimaryKey(autoGenerate = true)
    private long id;

    private String addressLine1;

    private String postalCode;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getAddressLine1() {
        return addressLine1;
    }

    public void setAddressLine1(String addressLine1) {
        this.addressLine1 = addressLine1;
    }

    public String getPostalCode() {
        return postalCode;
    }

    public void setPostalCode(String postalCode) {
        this.postalCode = postalCode;
    }
}

File: app/src/main/java/com/openclassrooms/realestatemanager/models/Address.kt
package com.openclassrooms.realestatemanager.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Address {

    @PrimaryKey(autoGenerate = true)
    var id: Long = 0

    var addressLine1: String? = null

    var postalCode: String? = null
}
